sdk/core/core-http/src/serviceClient.ts

```diff
@@ -469,12 +469,6 @@ export class ServiceClient {
 
       const options: RequestOptionsBase | undefined = operationArguments.options;
       if (options) {
-        if (options.customHeaders) {
```

Comment:
Why do we need to remove this from core-http? I expected this change to be in core-client only

Reply:
oops edited the wrong file 😓